### ISSUE TITLE
.snupkg Files created and pushed but NuGet not downloading them

### ISSUE THREAD

**Issue by LegendaryB** [NuGet/Home#9456]
Hey Guys,

me and a friend are developing a framework together and we are publishing nuget packages. We have now added creation of .snupkg files. They are also pushed to NuGet without problems. But when we use our packages Visual Studio is not able to step into the code.

I have already set "Enable just my Code" to false and checked if the NuGet symbol server is also referenced in the settings. But it does not work.

We have added the following to our csproj files to generate a .snupkg:
```
<IncludeSymbols>true</IncludeSymbols>
<SymbolPackageFormat>snupkg</SymbolPackageFormat>
```

I also tried clearing the nuget cache and restoring all packages. But the .snupkg files are never downloaded. Do I miss something here?

Thanks for your time and effort!

Kind Regards,
Daniel

**Comment by cristinamanum** [comment]
Hi @LegendaryB,

The snupkg are not downloaded. The Visual Studio debugger will download the matching pdbs in the debug session. 
In order to diagnose the issue, would you mind adding the below information:
1. package id and version that you try to debug
2. the Visual Studio version that you use

**Comment by LegendaryB** [comment]
Hey @cristinamanum ,

1. CodeMonkeys.Navigation.Xamarin.Forms - version 2.0.4
2. Visual Studio Professional 16.5.4

I also looked at the modules window to verify that the symbols are loaded. But it says they are not loaded. Maybe I really miss something here

**Comment by LegendaryB** [comment]
Any tips / ideas?

Our project can be found here: https://github.com/UltimateCodeMonkeys/CodeMonkeysMVVM/tree/master

**Comment by donnie-msft** [comment]
Is this still an issue? It sounds to me like your symbol sources aren't configured correctly with the **NuGet.org Symbol Server**.

You may check the VS guide on setting these up: scroll down to the section, [To specify symbol locations and loading options: ](https://docs.microsoft.com/en-us/visualstudio/debugger/specify-symbol-dot-pdb-and-source-files-in-the-visual-studio-debugger?view=vs-2019#symbol-file-locations-and-loading-behavior)

**Comment by LegendaryB** [comment]
Yeah it is still an issue. The snupkg is created and the pdb type is set to portable because nuget supports only portable format. I also cross checked all the settings in your link with my visual studio settings. But the symbols are still not loaded (also checked in the modules window). Do you have another idea? Or maybe could you try to debug into it? We have sample apps which can easily be downloaded and used. Would really appreciate it

**Comment by nkolev92** [comment]
Thank you for reporting this issue.
We prioritize investigations based on community engagement such as comments and up-votes. 
We might not be able to investigate further in the foreseeable future.

**Comment by moh-hassan** [comment]
Vs 2019 v16.x can not load the snupkg.
The only way to use .snupkg is to download the snupkg ->unzip -> add its path to server symbols.

This [link: Symbol file locations and loading behavior](https://docs.microsoft.com/en-us/visualstudio/debugger/specify-symbol-dot-pdb-and-source-files-in-the-visual-studio-debugger?view=vs-2019#symbol-file-locations-and-loading-behavior) can't help.
What is the official URL of snupkg symbol server?
